service/models.py

```diff
@@ -27,14 +27,17 @@ class YourResourceModel(db.Model):
 
     # Table Schema
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(63))
+    name = db.Column(db.String(63), nullable=False)
+    email = db.Column(db.String(63), nullable=False)
+    address = db.Column(db.String(256), nullable=False)
+    phone_number = db.Column(db.String(63))
```

Comment:
I think this needs to be less characters, phone numbers are only 15 characters at most...evan...